### ISSUE TITLE
[feature] Support for STM32U073

### ISSUE THREAD

**Issue by kevlar700** [stlink-org/stlink#1436]
Is there any kind of expected support timeline for STM32U073 in either stlink or openocd. I use Ada and need to connect gdb via something. Thank You.

- [x] Programmer/board type: STLINK V3 MINIE
- [x] Operating system and version: Linux Debian 12 Bookworm (Devuan actually but the same thing really)
- [x] STM32U073

**Comment by Nightwalker-87** [comment]
No, sorry - not in the foreseeable future, unless any volunteer is going to take action here.

**Comment by Ant-ON** [comment]
@kevlar700 U073 has similar flash peripheral to C0xx chips . You can create new config for U073 based on C0xx config.

**Comment by Nightwalker-87** [comment]
A first attempt can be found here: https://github.com/markxoe/stlink/commit/82741108113cd3af222a8ec4ed3ba316eb33cbfc.
However the device type still needs to be defined.
I can't say though, if additional changes are necessary for these devices.
